tests/forms_tests/field_tests/test_urlfield.py

```diff
@@ -136,7 +137,10 @@ def test_urlfield_strip_on_none_value(self):
         self.assertIsNone(f.clean(None))
 
     def test_urlfield_unable_to_set_strip_kwarg(self):
-        msg = "__init__() got multiple values for keyword argument 'strip'"
+        if PYPY:
+            msg = "got multiple values for keyword argument 'strip'"
+        else:
+            msg = "__init__() got multiple values for keyword argument 'strip'"
```

Comment:
I would remove `__init__()` from the message instead:
```suggestion
        msg = "got multiple values for keyword argument 'strip'"
```